process-migration-service/src/main/java/org/kie/processmigration/model/Execution.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.kie.processmigration.model;
+
+import java.net.URI;
+import java.time.LocalDateTime;
```

Comment:
If you care about timezones, the risk of pods running in different regions and your logic being sensitive to the actual timestamp, consider using DateTime instead of LocalDateTime.